src/components/select/multi-select/multi-select.stories.mdx

```diff
@@ -155,6 +155,38 @@ import { MultiSelect, Option } from "carbon-react/lib/components/select";
   </Story>
 </Preview>
 
+### Required
+
+You can use the `required` prop to indicate if the field is mandatory.
+<Preview>
+  <Story name="required" parameters={{ info: { disable: true }}} >
+   <MultiSelect
+      name='required-select'
+      id='required-select'
+      label='Foreground Color'
+      onOpen={ action('onOpen') }
+      onChange={ action('onChange') }
+      onClick={ action('onClick') }
+      onFocus={ action('onFocus') }
+      onBlur={ action('onBlur') }
+      onKeyDown={ action('onKeyDown') }
```

Comment:
there is no need to trigger all these actions as they could be already tested in the `basic` story.